lib/beaker-answers/versions/version20162.rb

```diff
@@ -9,5 +9,24 @@ class Version20162 < Version20161
     def self.pe_version_matcher
       /\A2016\.2/
     end
+
+    def generate_answers
+      the_answers = super
+
+      return the_answers if @options[:masterless]
+      console = only_host_with_role(@hosts, 'dashboard')
```

Comment:
Whoops, I'm totally wrong, because only `the_answers` are returned, none of the hosts by role. 😧  Please disregard the comment above.